docs/backup-restore.md

```diff
@@ -0,0 +1,101 @@
+
+### Requirements
+
+* You must wait at least 10 sec after pausing to create a backup.
+* You must be able to identify a common restore point for the Ledger and Auditor If you use Auditor.
+
+### Pause
+
+The `PAUSE` command helps you to create a transactionally consistent backup for non-transactional databases. It helps to pause the Scalar DL service.
+
+```console
+docker run -it --rm ghcr.io/scalar-labs/scalar-admin:<version> -c PAUSE -s <SRV_Service_URL>
+```
+
+### Unpause
+
+The `UNPAUSE` command helps you start the ledger after creating the backup.
+
+```console
+docker run -it --rm ghcr.io/scalar-labs/scalar-admin:<version> -c UNPAUSE -s <SRV_Service_URL>
+```
+
+
+## Create Backup
+
+This section shows how to create a transactionally-consistent backup for Scalar DL.
+
+### Cassandra
+
+Since Cassandra has a built-in replication mechanism, we don't always need a transactionally-consistent backup to recover a Cassandra cluster.
+
+For example, if replication is properly set to 3 and only the data of one of the nodes in a cluster is lost, we don't need a transactionally-consistent backup because the node can be recovered with a normal (transactionally-inconsistent) snapshot and the repair mechanism.
+However, if the quorum of nodes of a cluster loses their data, we need a transactionally-consistent backup to restore the cluster to a certain transactionally-consistent point.
+
+To minimize mistakes when doing backup operations, it is recommended to use [Cassy](https://github.com/scalar-labs/cassy).
+
+Cassy is integrated with `scalar-admin`, so it pauses the scalar DL service to create a transactionally-consistent cluster-wide backup.
+Please see [the doc](https://github.com/scalar-labs/cassy/blob/master/docs/getting-started.md#take-cluster-wide-consistent-backups) for more details.
+
+### JDBC
+
+You can take a backup with your favorite way for JDBC databases.
+One requirement for backup in Scalar DL on JDBC databases is that backups for all the Scalar DL managed tables (including the coordinator table) need to be transactionally-consistent or automatically recoverable to a transactionally-consistent state.
+That means that you need to create a consistent snapshot by dumping all tables in a single transaction. For example, you can use `mysqldump` command with `--single-transaction` option in MySQL and `pg_dump` command in PostgreSQL to achieve that.
+
+### Cosmos DB
+
+Cosmos DB continuous mode backup creates Point In Time Restore (PITR), it is recommended to use in production environment.
+
+The easiest way to take a transactionally-consistent backup for Scalar DL on Cosmos DB is to pause the Scalar DL service using the [How to create transactionally consistent backup](#how-to-create-transactionally-consistent-backup) steps. You can use the middle value of the pause time as a restore point.
+
+### DynamoDB
+
+ScalarDL schema loader enables Point-in-time recovery (PITR) for each table in a DynamoDB.
```

Comment:
```suggestion
ScalarDL schema loader enables Point-in-time recovery (PITR) for each table in a DynamoDB by default.
```

Reply:
Thank you for your suggestion.
Document updated 8ba311f